core/tests/protractor/userJourneys.js

```diff
@@ -22,7 +22,10 @@ var general = require('../protractor_utils/general.js');
 var users = require('../protractor_utils/users.js');
 var workflow = require('../protractor_utils/workflow.js');
 var LibraryPage = require('../protractor_utils/LibraryPage.js');
-
+var CreatorDashboardPage =
```

Comment:
Could we alphabetize the pages to keep it tidy? (E.g. AdminPage, CollectionEditorPage, etc.)

Reply:
Sure!